docs/eog/eps-risk-management-assessment.rst

```diff
@@ -2,21 +2,21 @@
 
 Risk Assessment
 ===============
-At the time of evaluation, the risks identified should be ranked by their likelihood and impact. The likelihood refers to the probability of risks actually happening, while impact is a score based on the most likely scenario and its ramifications after a potential risk happens. 
+At the time of evaluation, the risks identified should be ranked by their likelihood and impact. The likelihood refers to the probability of risks actually happening, while impact is the consequences if a potential risk happened.
```

Comment:
'if a potential risk happens.'

Reply:
made the update - see commit below - https://github.com/softwaresaved/event-organisation-guide/pull/75/commits/9f4eb537e07f18159ff4951a2b5ccc402cd0744b